### ISSUE TITLE
[BUG] SVG style fails this rule: Accessible name does not match or contain the visible label text

### ISSUE THREAD

**Issue by francesco-mariani** [IBMa/equal-access#411]
## What tool(s) are you using?

- [ ] `accessibility-checker-engine`
- [x] `accessibility-checker`
- [ ] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `boilerplates`
- [ ] `other`

## Detailed description

Accessibility Checker gives this violation
> Accessible name does not match or contain the visible label text

against
```
<button aria-label="text" type="button">
    <svg alt="text" focusable="false">
        <defs><style>.cls-1{fill:none;}</style></defs>
        <rect class="cls-1" width="32" height="32"></rect>
        <title>text</title>
    </svg>
</button>
```
due to the presence of `.cls-1{fill:none;}` inside the defs tag.

**Comment by tombrunet** [comment]
Adding code example: https://codesandbox.io/s/issue-411-j0it0?file=/index.html

**Comment by philljenkins** [comment]
Work in #2055 may close this issue

**Comment by shunguoy** [comment]
the issue was fixed in the engine refactoring #2055
